### PR TITLE
Finished some tasks

### PR DESCRIPTION
**Pull request by IkeVoodoo**
#166 | Done
#125 | Done
#160 | Done

### CODE REVIEW

File: src/main/java/com/atherys/towns/command/plot/BordersCommand.java [hunk 1]
```diff
@@ -19,19 +19,13 @@
...
-                GenericArguments.bool(Text.of("border"))
-        };
-    }
+public class BordersCommand implements PlayerCommand {
 
     @Nonnull
     @Override
     public CommandResult execute(@Nonnull Player source, @Nonnull CommandContext args) throws CommandException {
-        AtherysTowns.getInstance().getPlotBorderFacade().setPlayerViewBorderStatus(source, args.<Boolean>getOne("border").orElse(false));
+        AtherysTowns.getInstance().getPlotBorderFacade().setPlayerViewBorderStatus(
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/command/plot/BordersCommand.java:27]: Just remove the parameter from the method signature and grab the status in the facade's method.

File: src/main/java/com/atherys/towns/command/town/SetTownJoinableCommand.java [hunk 2]
```diff
@@ -19,19 +19,13 @@
...
-        };
-    }
+public class SetTownJoinableCommand implements PlayerCommand {
 
     @Nonnull
     @Override
     public CommandResult execute(@Nonnull Player source, @Nonnull CommandContext args) throws CommandException {
         AtherysTowns.getInstance().getTownFacade().setPlayerTownJoinable(
-                source, args.<Boolean>getOne("joinable").get()
+                source, !AtherysTowns.getInstance().getTownFacade().getPlayerTown(source).isFreelyJoinable()
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/command/town/SetTownJoinableCommand.java:28]: Same for this.

File: src/main/java/com/atherys/towns/command/town/SetTownMobsCommand.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+@Aliases("mobs")
+@Description("Enables/Disables Hostile Mob Spawning in your Town.")
+@Permission("atherystowns.town.mobs")
+public class SetTownMobsCommand implements PlayerCommand {
+
+    @Nonnull
+    @Override
+    public CommandResult execute(@Nonnull Player source, @Nonnull CommandContext args) throws CommandException {
+        AtherysTowns.getInstance().getTownFacade().setPlayerTownMobs(source,
+                !AtherysTowns.getInstance().getTownFacade().getPlayerTown(source).isMobs());
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/command/town/SetTownMobsCommand.java:24]: And this.

File: src/main/java/com/atherys/towns/command/town/SetTownPvpCommand.java [hunk 4]
```diff
@@ -10,27 +10,19 @@
...
-        };
-    }
-
+public class SetTownPvpCommand implements PlayerCommand {
     @Nonnull
     @Override
     public CommandResult execute(@Nonnull Player source, @Nonnull CommandContext args) throws CommandException {
-        AtherysTowns.getInstance().getTownFacade().setPlayerTownPvp(source, args.<Boolean>getOne("pvp").orElse(false));
+        AtherysTowns.getInstance().getTownFacade().setPlayerTownPvp(source,
+                !AtherysTowns.getInstance().getTownFacade().getPlayerTown(source).isPvpEnabled());
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/command/town/SetTownPvpCommand.java:25]: And this.

File: src/main/java/com/atherys/towns/facade/PlotFacade.java [hunk 5]
```diff
@@ -111,6 +112,14 @@ public TownPlot getPlotAtPlayer(Player player) throws TownsCommandException {
                 new TownsCommandException("You are not standing in a plot."));
     }
 
+    public boolean checkHostileSpawningAtLocation(Location<World> location) {
+        AtomicBoolean spawn = new AtomicBoolean(true);
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/facade/PlotFacade.java:116]: You don't need to use an atomic here. Just ` return .map(plot -> plot.getTown().isMobs).orElse(false)`.

File: src/main/java/com/atherys/towns/facade/TownFacade.java [hunk 6]
```diff
@@ -264,6 +264,13 @@ public void setPlayerTownJoinable(Player player, boolean joinable) throws TownsC
         townsMsg.info(player, "Your town is now ", joinable ? "freely joinable." : "not freely joinable.");
     }
 
+    public void setPlayerTownMobs(Player playerb, boolean mobs) throws TownsCommandException {
```
Comment (LJNIC) [inline src/main/java/com/atherys/towns/facade/TownFacade.java:267]: Why `playerb`?

**Review by fingoltin — CHANGES_REQUESTED** [review]
Looks good aside from a few minor things. Nice work.

**Pull request merged**